pkg/tsdb/azuremonitor/azure-resource-graph-datasource.go

```diff
@@ -173,15 +173,37 @@ func (e *AzureResourceGraphDatasource) executeQuery(ctx context.Context, query *
 	if err != nil {
 		return dataResponseErrorWithExecuted(err)
 	}
-	if frame.Meta == nil {
-		frame.Meta = &data.FrameMeta{}
+
+	azurePortalUrl, err := getAzurePortalUrl(dsInfo.Settings.CloudName)
```

Comment:
`dsInfo.Settings.CloudName` is not proper way to name of the cloud.
It won't work for managed identities.

I have open PR which exposes a new field `dsInfo.Cloud` which will be always correct for both Managed Identity and App Registration.

You may wait a couple of days until that PR is merged.

#36284